src/gluonts/torch/model/lightning_util.py

```diff
@@ -18,5 +18,6 @@
 
 def has_validation_loop(trainer: pl.Trainer):
     if version.parse(pl.__version__) < version.parse("2.0.0"):
+        assert hasattr(trainer._data_connector, "_val_dataloader_source")
```

Comment:
We have changed the dependency range for lightning to `>=2.0`, can we just drop this check completely?

Reply:
done